target/linux/ipq806x/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -32,6 +32,13 @@ case "$FIRMWARE" in
 			caldata_extract "0:ART" 0x1000 0x2f20
 		fi
 		;;
+	extreme,ap3935i)
+		if [ -e /lib/firmware/ath10k/QCA99X0/hw2.0/board-5g-precal.bin ]; then
```

Comment:
What's the purpose of the if? In which situations would he have no such "if" here? What would be the consequences?

Reply:
The logic is that we don't want to create the symlink if it exists already, or if the user has added in a different precal file

Reply:
Changed the logic to just cp always

Reply:
Be aware of line 3. Is that what you wanted?

Reply:
Yeah, that works fine.  If it exists, don't worry about copying it.  Thanks for the heads up on line 3!